htmd/simlist.py

```diff
@@ -12,6 +12,7 @@
 from htmd.molecule.molecule import Molecule
 from joblib import Parallel, delayed
 from htmd.progress.progress import ProgressBar
+import mdtraj as md
 import logging
```

Comment:
Yes, I noticed after. I just had some problems at the start finding everything so for test purposes I wrote some code around the Molecule class just to get things working. However, you do not seem to support writing out .rst files.. So I added a _writeRST() method to Molecule.